sssd_test_framework/utils/tools.py

```diff
@@ -733,3 +740,81 @@ def __exec(self, cls, cmd: str, name: str | int, service: str | None = None) ->
             return None
 
         return cls.FromOutput(command.stdout)
+
+
+class SSHKeysUtils(MultihostUtility[MultihostHost]):
+    """
+    SSH key utilities on remote host.
+    """
+
+    def __init__(self, host: MultihostHost, fs: LinuxFileSystem) -> None:
+        super().__init__(host)
+        self.host: MultihostHost = host
+        self.fs: LinuxFileSystem = fs
+        self.user: str | None = None
+        self.path: str | None = None
+        self.file: str | None = None
+
+    def keygen(
+        self,
+        user: str,
+        path: str,
+        file: str = "id_rsa",
+        cipher: str = "rsa",
```

Comment:
Is there any use for also supporting a `bits` option as well?  Or it can just be handled if needed with `args`?

Reply:
That can be handled by args.